awscli/clidriver.py

```diff
@@ -656,10 +657,15 @@ def invoke(self, service_name, operation_name, parameters, parsed_globals):
             value is returned.
 
         """
+        if hasattr(parsed_globals, 'read_timeout'):
```

Comment:
Which test fails? Maybe the test needs to get updated. The global parsing should add ``None`` for it when missing.